src/components/Modal/Modal.vue

```diff
@@ -13,7 +13,10 @@
         aria-labelledby="header"
         aria-describedby="modalDescription"
         :style="{'width': width}"
-        class="relative bg-white flex flex-col overflow-y-auto shadow rounded-lg p-5 max-h-5/6"
+        :class="[
+          'relative bg-white flex flex-col overflow-y-auto shadow rounded-lg p-5 max-h-5/6',
+          {'!p-0' : noPadding}
```

Comment:
Is the `!` really needed?  I would expect it to override the previous `p-5` without it but I guess I'm not sure that it's added in order after the previous line.

Reply:
it is necessary